example/hello_object_detection_ssd_node/README.md

```diff
@@ -0,0 +1,77 @@
+# A Node.js example of object detection ssd using inference-engine-node
+
+### Install
+
+```sh
+$ npm install
+```
+
+### Usage
+
+```sh
+$ node main.js --help
+
+Hello Object Detection SSD
+
+  An example of object detection ssd using inference-engine-node.
+
+Options
+
+  -h, --help                Show this help message and exit.
+  -m, --model string        Required. Path to an .xml file with a trained model.
+  -i, --image string        Required. Path to an image file.
+  -d, --device string       Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL, MYRIAD
+                            or HETERO: is acceptable. Default value is CPU.
+  -n, --iterations number   Optional. The number of iterations for inference. Default value is 1.
+  -t, --threshold number    Optional. The minimal probability of the visibility object. Default value is 0.5.
+  -s, --sync                Optional. Specify to inference synchronously or asynchronously. Default value
+                            is false.
+```
+
+### Run
+
+For example on Windows, run SqueezeNet on CPU plugin for 10 iterations:
+```sh
+$ node main.js -m ..\..\models\mobilinetssd\FP32\mobilinetssd.xml -i test.png -d CPU -n 10
+-------------------------------------------
+Check inference engine version:
+  API version: 2.1
+  Build: 37988
+  Description: API
+-------------------------------------------
+Start to create network from /home/user/Developer/resources/models/IR/OD/ssd_mobilenet_v1_coco/ssd_mobilenet_v1_coco.xml.
+Succeeded: read network took 37.40 ms.
+Network name: Function_0
+Input[0]:
+  name: image_tensor
+  precision: fp32
+  layout: nchw
+  dims: [1,3,300,300]
+Output[0]:
+  name: DetectionOutput
+  precision: fp32
+  layout: nchw
+  dims: [1,1,100,7]
+Change input layout to 'nhwc' and precision to 'u8'.
+-------------------------------------------
+Start to read image from test.jpg.
+Succeeded.
+Resize image from (3120, 4160) to (300, 300).
+-------------------------------------------
+Check CPU plugin version:
+  Deivce Name: CPU
+  API version: 2.1
+  Build: 37988
+  Description: MKLDNNPlugin
+Start to load network to CPU plugin.
+Succeeded: load network took 331.25 ms.
+-------------------------------------------
+Start to infer asynchronously for 1 iterations.
+Succeeded: the average inference time is 13.86 ms.
+           the throughput is 72.13 FPS.
```

Comment:
On my system, it shows
```
Error: ENOENT: no such file or directory, open 'C:\Users\nhu\code\inference-engine-node\models\mobilinetssd\FP32\mobilenetssd.labels'
```

Do you want to add lables?

Reply:
Fixed in commit: "Review fixes"